upgrading/topics/keycloak/changes.adoc

```diff
@@ -2,6 +2,27 @@
 
 === Migrating to 11.0.0
 
+==== Upgrade to Wildfly 20
+
+The {project_name} server was upgraded to use Wildfly 20 as the underlying container. This does not directly involve any
+specific {project_name} server functionality, but there are few changes related to the migration, which worth mentioning.
```

Comment:
(Nitpick) <pre> s/which worth mentioning/which are worth mentioning/g </pre> ?

Reply:
Will update, Thanks

Reply:
This wasn't updated yet (maybe because I marked the comment as hidden?) When updating the point below, could you fix this too?

Reply:
Updated. Thanks for the catch!